CHANGELOG.md

```diff
@@ -144,6 +144,7 @@ Shipping methods can be removed by the user after it has been assigned to a chec
 - Fix seo field to accept null value - #13512 by @ssuraliya
 - Add missing descriptions to payment module - #13546 by @devilsautumn
 - Fix `NOTIFY_USER` allow to create webhook with only one event - #13584 by @Air-t
+- Fix error in variant available stock calculation - 13593 by @awaisdar001
```

Comment:
@awaisdar001 Could you rebase your PR and move this entry to 3.16 section? 3.15 scope is already closed and will be released soon, any new changes from now on go to 3.16.

Reply:
@maarcingebala Done, thanks for suggesting me the changes.